barsys/templates/email/bank_account_details.html.html

```diff
@@ -21,5 +21,11 @@
         <th>Payment reference</th>
         <td>{{ pybarsys_preferences.EMAIL.BANK_ACCOUNT_PAYMENT_REFERENCE }} {{ recipient.display_name|clean_str }}</td>
     </tr>
+    {% if invoice.comment != "" %}
+    <tr>
+        <th>Comment</th>
+        <td>{{ invoice.comment|urlcheck|safe }}</td>
```

Comment:
The name of `urlcheck` changed (same for the other files)

Reply:
yeah was just fixing it hahahah didn't know how I can do everything in 1 commit on the github site